decentraland/unity-renderer
# fix: gltf importer texture scale hiccup

## What does this PR change?

This PR reimplements our texture resizing code to happen entirely GPU side. This code is used by the GLTF importer (and will potentially be used for more optimizations). In the previous implementation, memory had to be copied between GPU and CPU, causing big noticeable hiccups.

## How to test the changes?



1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/gltf-importer-texture-scale-hiccup&ENV=org&NO_ASSET_BUNDLES
2. No new texture artifacts should be observed for models that use textures that go up of 1024x1024

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
